### ISSUE TITLE
Math in list duplicated with every keystroke

### ISSUE THREAD

**Issue by kylebarron** [atom-community/markdown-preview-plus#401]
![gif](https://user-images.githubusercontent.com/15164633/41048969-e0f1e2ba-697d-11e8-9382-c73dcb76f86b.gif)

text in editor at start:
```
# Hello world

1. this is a list
2.
    $$aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa + bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb = cccccccccccc$$

    $$hello hello_ht = b0 + b1hello_world_hi if hello_world_hi==3 | hello_world_hi==4, hello hello(hello_world_hi)$$



this is math
```

Package configuration:
```cson
  "markdown-preview-plus":
    grammars: [
      "source.gfm"
      "source.litcoffee"
      "text.html.basic"
      "text.md"
      "text.plain"
      "text.plain.null-grammar source.pweave.md"
    ]
    markdownItConfig:
      breakOnSingleNewline: true
      useLazyHeaders: false
    mathConfig:
      enableLatexRenderingByDefault: true
    pandocConfig:
      pandocBibliography: true
      pandocRemoveReferences: false
    renderer: "pandoc"
    useGitHubStyle: true
```

**Comment by kylebarron** [comment]
Thanks!

**Comment by lierdakil** [comment]
Thanks for reporting! This is due to a flaky fix for #386 which apparently confused MathJax. I'll try to push out a release this week.
